com.unity.perception/Runtime/GroundTruth/InstanceIdToColorMapping.cs

```diff
@@ -142,17 +148,52 @@ public static bool TryGetColorFromInstanceId(uint id, out Color32 color)
         }
 
         /// <summary>
-        /// Retrieve the ID associated with the passed in color. If the passed in color is black, or has an alpha channel
-        /// of 0, or has an alpha channel of value of 255, but the RGB values are not associated with a known color, this
-        /// service will return false, and the out id will be set to 0.
+        /// Retrieve the color that is mapped to the passed in ID. If the ID is 0 or 255 the returned color will be black.
+        /// </summary>
+        /// <param name="id">The ID of interest.</param>
+        /// <returns>The color associated with the passed in ID, or black if no associated color exists.</returns>
+        /// <exception cref="IndexOutOfRangeException">Thrown if the passed in ID is greater than the largest supported ID <see cref="MaxId"/></exception>
+        public static Color32 GetColorFromInstanceId(uint id)
+        {
+            if (id > MaxId) throw new IndexOutOfRangeException($"Passed in index: {id} is greater than max ID: {MaxId}");
```

Comment:
This code duplicates TryGetColorFromInstanceId. Would suggest just making this a wrapper around that method.

Reply:
yeah, not sure what I was thinking there.

Reply:
fixed